test/util/spec.helpers.js

```diff
@@ -0,0 +1,117 @@
+var helpers = require('../../lib/util/helpers');
+
+describe('helpers', function () {
+
+    describe('getPreviousSteps', function () {
+        var getPreviousSteps, steps;
+
+        beforeEach(function () {
+            getPreviousSteps = helpers.getPreviousSteps;
+        });
+
+        it('is a function', function () {
+            getPreviousSteps.should.be.a('function');
+        });
+
+        it('returns an array', function () {
+            getPreviousSteps().should.be.an('array');
+        });
+
```

Comment:
@gavboulton if that's the case then perhaps I'm misunderstanding the way require/sinon are working together when testing! Not too sure why we are using proxyquire in a lot of places if we can just stub and future require calls return the stubbed method, I guess proxyquire more appropriate for stubbing larger deps, like a whole controller.

How should we do this? Shall I rebase your changes into my branch, or replace this PR with a PR from your branch to master?